pkg/cvo/cvo.go

```diff
@@ -613,11 +636,26 @@ func versionString(update configv1.Update) string {
 }
 
 // currentVersion returns an update object describing the current known cluster version.
```

Comment:
it now returns a release but also has the new update logic

Reply:
Documented with f666975b -> 1316b58c.